docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml

```diff
@@ -1220,94 +1529,202 @@ ecl run thor findperson.ecl -I C:\MyECL\
                 <row>
                   <entry>--manifest</entry>
 
-                  <entry>Specify path to manifest file</entry>
+                  <entry>Specify path to the manifest file</entry>
                 </row>
 
                 <row>
-                  <entry>-checkDirty</entry>
+                  <entry>-g, --debug</entry>
 
-                  <entry>Causes eclcc to generate a warning for any attribute
-                  that has been modified (according to the output of git
-                  status). Use of this function requires that git be installed
-                  and available on the path.</entry>
+                  <entry>Enable debug symbols in generated code</entry>
                 </row>
 
                 <row>
-                  <entry>--nostdinc</entry>
+                  <entry>-checkDirty</entry>
 
-                  <entry>Do not include the current directory in -I</entry>
+                  <entry>This option reports any modified attributes using git
+                  status</entry>
                 </row>
 
                 <row>
-                  <entry>--fastsyntax</entry>
+                  <entry>-checkDirty</entry>
```

Comment:
This option seems to be repeated

Reply:
good catch!

Reply:
And the description on first of the two doesn't match other -checkDirty entries.  So the second description should win.